### ISSUE TITLE
Telegram column?

### ISSUE THREAD

**Issue by ente3000** [jariz/tabbie#114]
As Telegram has an API, I assume it would be easy (or at least easier than adding a WhatsApp column) to add a Telegram column. Would that be possible? It's the only feature I'm really missing. Great work otherwise!

**Comment by TheGhoul21** [comment]
what features do you  expect from a telegram column?

**Comment by ente3000** [comment]
An interface like Gmail showing unread messages in bold and read messages in ... normal

**Comment by TheGhoul21** [comment]
sounds fine and useful actually. No way to respond to messages though, right? That will be done after (let's say) clicking one thread in a _blank page

**Comment by ente3000** [comment]
Yep. There's no way to do that in the Gmail column either, so that's fine.
